### ISSUE TITLE
Issues with Docker Compose Instructions

### ISSUE THREAD

**Issue by theamazingnorad** [Steam-Headless/docker-steam-headless#137]
### Description

As a dual AMD GPU user on Arch under KDE 6.0, I struggled to get this working.   Three things I would recommend adding to the instructions. 

- The default port information is misleading.  The example .env uses 58083 yet the install guide references 8083.
- I had to go into 70-configure_xorg.sh to figure out why my x server was not running -- I was getting errors about Xauthority, but traced it down to not having a valid display device.  I'm not sure what the DISPLAY variable of 55 is trying to do, but I ended up adding FORCE_X11_DUMMY_CONFIG=TRUE to my environmental variables to get Xorg in the container to create a dummy display and start.  
- You have a commented out line in docker-compose to pass /dev/dri.  I'd suggest adding a blurb that, for multiple AMD GPU users, you need to pass the specific GPU you want.  So for me, /dev/dri/card1 and /dev/dri/renderD128.  At that point, only my AMD 6500 passes and not my 6950XT.  Confirmed working with vkvube from the container.  I was having issues with it trying to run things on the wrong GPU which was already bound to Wayland in my host.  

Anyways, these edits allowed me to get this working.  My wife can now game off my 6500 in the container from her Mac while I play off my 6950XT.  Hurray 5950X and 64 GB ram :)

**Comment by cRaZy-bisCuiT** [comment]
Thanks for the info @theamazingnorad ! Would you mind to open a PR and update the needed changes? Otherwise I could try to build on my Arch system as well and then open a PR. :)

**Comment by Josh5** [comment]
> ### Description
> 
> As a dual AMD GPU user on Arch under KDE 6.0, I struggled to get this working.   Three things I would recommend adding to the instructions. 
> 
> - The default port information is misleading.  The example .env uses 58083 yet the install guide references 8083.
> - I had to go into 70-configure_xorg.sh to figure out why my x server was not running -- I was getting errors about Xauthority, but traced it down to not having a valid display device.  I'm not sure what the DISPLAY variable of 55 is trying to do, but I ended up adding FORCE_X11_DUMMY_CONFIG=TRUE to my environmental variables to get Xorg in the container to create a dummy display and start.  
> - You have a commented out line in docker-compose to pass /dev/dri.  I'd suggest adding a blurb that, for multiple AMD GPU users, you need to pass the specific GPU you want.  So for me, /dev/dri/card1 and /dev/dri/renderD128.  At that point, only my AMD 6500 passes and not my 6950XT.  Confirmed working with vkvube from the container.  I was having issues with it trying to run things on the wrong GPU which was already bound to Wayland in my host.  
> 
> Anyways, these edits allowed me to get this working.  My wife can now game off my 6500 in the container from her Mac while I play off my 6950XT.  Hurray 5950X and 64 GB ram :) 

With the setup you have achieved,  are the 2 of you able to play at the same time?

**Comment by theamazingnorad** [comment]
> > As a dual AMD GPU user on Arch under KDE 6.0, I struggled to get this working.   Three things I would recommend adding to the instructions.
> > 
> > * The default port information is misleading.  The example .env uses 58083 yet the install guide references 8083.
> > * I had to go into 70-configure_xorg.sh to figure out why my x server was not running -- I was getting errors about Xauthority, but traced it down to not having a valid display device.  I'm not sure what the DISPLAY variable of 55 is trying to do, but I ended up adding FORCE_X11_DUMMY_CONFIG=TRUE to my environmental variables to get Xorg in the container to create a dummy display and start.
> > * You have a commented out line in docker-compose to pass /dev/dri.  I'd suggest adding a blurb that, for multiple AMD GPU users, you need to pass the specific GPU you want.  So for me, /dev/dri/card1 and /dev/dri/renderD128.  At that point, only my AMD 6500 passes and not my 6950XT.  Confirmed working with vkvube from the container.  I was having issues with it trying to run things on the wrong GPU which was already bound to Wayland in my host.
> > 
> > Anyways, these edits allowed me to get this working.  My wife can now game off my 6500 in the container from her Mac while I play off my 6950XT.  Hurray 5950X and 64 GB ram :)
> 
> With the setup you have achieved, are the 2 of you able to play at the same time?

Correct.  We wanted to play BG3, the Steam Deck struggles with this, and she was able to to Remote Play on her Macintosh from this docker container.  Note that, in order to get it working, I bound the docker container to its own network adapter so it had a unique IP address on the network.  Steam struggles to do Remote Play otherwise.  I've since moved on to a Windows VM with GPU passthough since she wanted to play some games that require Anti Cheat *sigh* 

I will try to spin this back up this weekend, however, while my wife is at work and make the PR on the instructions.   The real issue here is that the docker documentation is....not good for AMD GPUs.  I blame this primarily on the influence of CUDA and the primarily NVIDIA focus by enterprise users.   I'll reference folks to Linus' opinion on NVIDIA for further commentary :p

**Comment by andreasntr** [comment]
> I'm not sure what the DISPLAY variable of 55 is trying to do, but I ended up adding FORCE_X11_DUMMY_CONFIG=TRUE 

This worked for me as well, thanks!

**Comment by Josh5** [comment]
I am currently restructuring our issue tracker and retiring the legacy documentation queue in preparation for **Steam Headless V2**.

Many of these inquiries relate to hardware setup, compatibility checks, or configuration guidance:

If you need assistance getting your specific hardware/software configuration running, please join our [Discord Community](https://streamingtech.co.nz/discord) where our community is active and happy to help.

Following the release of Steam Headless V2, feel free to re-submit a **Compatibility Report** or a request to start supporting your OS/Hardware combination.

I am closing this issue for now. If you encounter a verified bug or regression after testing V2, please feel free to open a new issue.

Thanks!
